scripts/run-pallet-benchmarks.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+read -p 'Pallet name: ' pallet
+
+pallet_name=$(echo "$pallet" | tr '-' '_')
```

Comment:
```suggestion
PS3="Pallet name: "
select pallet_name in $(ls pallets | tr '-' '_'); do break; done 
 ```

If you're feeling fancy.